datafusion/sql/src/unparser/plan.rs

```diff
@@ -158,10 +158,12 @@ impl Unparser<'_> {
         }
 
         let mut twj = select_builder.pop_from().unwrap();
-        twj.relation(relation_builder);
+        twj.relation(relation_builder.clone());
```

Comment:
I don't understand why this needs to have a clone now 🤔

Reply:
Because `twj.relation()` takes ownership of the `relation_builder` causing it to move, so we can't borrow it again later.

The only thing I use the relation builder for is to retrieve the list of all the identifiers, so I could probably do that before the `twj.relation()` then just pass those like:

```
let valid_idents = select_builder.collect_valid_idents();
twj.relation();
```

which shouldn't require a clone.